Max height for heightmap

antzGames opened the pull request:
Added a `minMax` field to `HeightMapTab.kt`

If left blank, or set to zero (which would generate a flat terrain) then a default value positive 100 is set.
If user sets the minMax value to negative then the heightmap is inverted.

I think the `Terraformer` class does not allow negative `minHeight` so that is why I only implemented one min/max field.

See https://github.com/JamesTKhan/Mundus/issues/174#issuecomment-1567232389 for more info and screenshots.

Dgzt commented:
Looks good. I think a tooltip with some information would be useful like in settings. What do you think?

JamesTKhan commented:
I agree a tooltip would be preferred if feasible, to explain the  behavior like when the value is negative.

antzGames commented:
Ok I added more detailed information:

![image](https://github.com/JamesTKhan/Mundus/assets/10563814/ca267ee4-dce8-4bb6-9088-626e8d9a6406)

So I removed negative values.   If the user enters a negative value, then I make that value positive.  Empty or zero values defaults to 100.

Let me know if you have any more suggestions.

JamesTKhan commented:
Looks good on my end, thank you!
